docs/source/examples/array-random.rst

```diff
@@ -0,0 +1,12 @@
+Creating random arrays
+======================
+
+In a simple case, we can create arrays with random data using the `da.random`
+module.
+
+.. code-block:: python
+
+    >>> import dask.array as da
+    >>> x = da.random.normal(0, 1, size=(100000,100000), chunks=(10000, 10000))
```

Comment:
A chunk size of `(1000, 1000)` is probably much more safe here.  A chunk of `(10000, 10000)` of `float64`s is 800 MB.  We often have 10-20 chunks floating about so this can become dangerous.  Shooting for chunks in the 1-100MB range is generally safer.